_includes/page__meta.html

```diff
@@ -0,0 +1,26 @@
+{% assign document = post | default: page %}
```

Comment:
document is still generic, I always like self contained variables, what about page_meta as variable?

Reply:
> document is still generic, I always like self contained variables, what about page_meta as variable?

Nevermind, shouldn't comment at 1am. good work. Document is the current `document` so it's good for me;